ui/admin/app/routes/scopes/scope/sessions.js

```diff
@@ -116,4 +117,13 @@ export default class ScopesScopeSessionsRoute extends Route {
   clearAllFilters() {
     this.status = [];
   }
+
+  /**
+   * refreshes session data.
+   */
+  @action
+  @loading
+  async refreshSessions() {
```

Comment:
We are defining `async` for a reason? no context on `ember-loading` so apologize is the question is too obvious.

Reply:
I thought this function is a good candidate to use the [@loader helper](https://github.com/hashicorp/boundary-ui/tree/main/addons/core#loading-helper) in case it consumes a lot of time.

Reply:
Awesome! thanks for the info :smile:

Reply:
Not sure if it would work, but could you call `refresh` directly?  The only requirement would be that it be annotated as an action.

```js
@action
refresh() {
  return super.refresh(...arguments);
}
```

Reply:
updated!